src/iuse.cpp

```diff
@@ -939,6 +940,14 @@ int iuse::flu_vaccine( player *p, item *it, bool, const tripoint & )
     return it->type->charges_to_use();
 }
 
+int iuse::prophylactic_antivenom( player *p, item *it, bool, const tripoint & )
+{
+    p->add_msg_if_player( m_good,
+                          _( "You feel resistant to venoms." ) );
+    p->add_effect( effect_took_prophylactic_antivenom, 1_weeks, true );
```

Comment:
```suggestion
    p->add_effect( effect_prophylactic_antivenom, 1_weeks, true );
```
(As you have almost certainly also noticed...)